compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -99,6 +99,22 @@ impl Attribute {
         }
     }
 
+    pub fn is_path(&self, name: &[Symbol]) -> bool {
```

Comment:
The name here is a bit weird, since it's inconsistent with `is_word`, which only allows `#[word]` and not `#[word]`. Maybe let's name this `path_matches` or something?